packages/common/src/types/scheduler.ts

```diff
@@ -76,9 +77,15 @@ export type ChartScheduler = SchedulerBase & {
     savedChartUuid: string;
     dashboardUuid: null;
 };
+
+export const isDashboardScheduler = (
+    scheduler: Scheduler | CreateSchedulerAndTargets,
+): scheduler is DashboardScheduler => scheduler.dashboardUuid !== undefined;
+
 export type DashboardScheduler = SchedulerBase & {
     savedChartUuid: null;
     dashboardUuid: string;
+    filters?: DashboardFilterRule[];
```

Comment:
Do we need to update the API schema?

Reply:
thanks for reminding me, pushed 👍

Reply:
hmm it didn't make changes to the routes 🤔